examples/external_ros_resources/drake_example_pendulum/src/drake_example_pendulum.cc

```diff
@@ -0,0 +1,87 @@
+/*****************************************************************************
+ * Copyright (c) 2017-2021, Massachusetts Institute of Technology.
+ * Copyright (c) 2017-2021, Toyota Research Institute.
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *****************************************************************************/
+
+#include <iostream>
+
+#include <drake/geometry/drake_visualizer.h>
+#include <drake/multibody/parsing/parser.h>
+#include <drake/multibody/plant/multibody_plant.h>
+#include <drake/multibody/tree/revolute_joint.h>
+#include <drake/systems/analysis/simulator.h>
+#include <drake/systems/framework/diagram_builder.h>
+
+int main(int argc, char** argv) {
+  drake::systems::DiagramBuilder<double> builder;
+  auto [pendulum_plant, scene_graph] =
+      drake::multibody::AddMultibodyPlantSceneGraph(&builder, 0.0);
+  auto parser = drake::multibody::Parser(&pendulum_plant, &scene_graph);
+
+  // Populate from AMENT_PREFIX_PATH environment variable to find URDF files and
+  // their resources, such as meshes.
+  parser.package_map().PopulateFromEnvironment("AMENT_PREFIX_PATH");
```

Comment:
@IanTheEngineer @EricCousineau-TRI any interest in in this example?

A lot has changed since this PR opened. I think now we'd put this in a folder in [`drake_ros_examples`](https://github.com/RobotLocomotion/drake-ros/tree/develop/drake_ros_examples/examples). The key piece of information appears to be calling [`PackageMap::PopuldateFromEnvironment`](https://drake.mit.edu/doxygen_cxx/classdrake_1_1multibody_1_1_package_map.html#ab17a29ee7c846aaca8f0d50248a42068) with `AMENT_PREFIX_PATH`.

Reply:
Yes, let's revive this once #207 lands!